index.html

```diff
@@ -0,0 +1,126 @@
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:200&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Apple</title>
+  </head>
+  <body>
+    <section class="principal">
+      <nav>
+        <ul class="top-nav">
+          <li class="grid left-nav-item grey-grad">
+            <a class="grid align-center m5" href="#"><i class="fa fa-apple"></i></a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Store</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Mac</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPod</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPhone</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iPad</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">iTunes</a>
+          </li>
+          <li class="grid nav-item grey-grad">
+            <a class="grid align-center m5" href="#">Support</a>
+          </li>
+          <li class="grid right-nav-item">
+            <i class="fa fa-search fa grey-grad"><input type="text" class="topics grey-grad m5"></i>
+          </li>
+        </ul>
+      </nav>
+      <section class="main-text">
+        <a href="#">What will your verse be?</a>
+        <p>
+      Everyone has something to share. See how people are using
+       iPad in their unique ways to add to the world's story.
+        </p>
+      </section>
+    </section>
+    <aside class="promos">
+      <section class="grid2 promos-nav">
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+          </section>
+          <section>
+            <img class="w100" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg"  alt="Supplier Responsibility The latest results of our worldwide efforts.">
```

Comment:
The **```<img>tags```** should be wrapped with an **``` <a>tag```** with **```href='#'```** . When you [click](https://web.archive.org/web/20140301004610/http://www.apple.com/) on the images, they should take you to another webpage.